package.json

```
@@ -18,12 +18,17 @@
     "ci-check": "npm-run-all prettier:diff --parallel lint tsc lint-heading-ids rss deadlinks",
     "tsc": "tsc --noEmit",
     "start": "next start",
-    "postinstall": "is-ci || husky install .husky",
+    "postinstall": "yarn --cwd eslint-local-rules install && is-ci || husky install .husky",
     "check-all": "npm-run-all prettier lint:fix tsc rss",
     "rss": "node scripts/generateRss.js",
     "deadlinks": "node scripts/deadLinkChecker.js",
+<<<<<<< HEAD
     "textlint": "cd textlint && yarn --frozen-lockfile && yarn textlint",
     "textlint-staged": "cd textlint && yarn --frozen-lockfile && yarn textlint-staged --"
+=======
+    "copyright": "node scripts/copyright.js",
+    "test:eslint-local-rules": "yarn --cwd eslint-local-rules test"
+>>>>>>> f8c81a0f4f8e454c850f0c854ad054b32313345c
```

ja独自のスクリプトは先頭 (`scripts {`直下) に追加した方がコンフリクトしにくい気がしましたが現状のまま末尾にしました

```suggestion
    "copyright": "node scripts/copyright.js",
    "test:eslint-local-rules": "yarn --cwd eslint-local-rules test",
    "textlint": "cd textlint && yarn --frozen-lockfile && yarn textlint",
    "textlint-staged": "cd textlint && yarn --frozen-lockfile && yarn textlint-staged --"
```